### ISSUE TITLE
-fstack-protector-strong not supported by compiler

### ISSUE THREAD

**Issue by tinxx** [nning/linux-grsec#5]
Despite `gcc` not protesting when compiling a sample program with the option `-fstack-protector-strong`, while building *linux-grsec* it complains about the aforementioned option.

Same happens when building from Arch Linux `community` repository.

I don't know if I didn't realize it before or if it's a new error.

This issue refers (at least) to 0d6b88f (linux 4.5.7, grsec 3.1/201606110914).

```
patching file tools/virtio/linux/uaccess.h
patching file virt/kvm/kvm_main.c
  HOSTCC  scripts/basic/fixdep
  HOSTCC  scripts/kconfig/conf.o
  SHIPPED scripts/kconfig/zconf.tab.c
  SHIPPED scripts/kconfig/zconf.lex.c
  SHIPPED scripts/kconfig/zconf.hash.c
  HOSTCC  scripts/kconfig/zconf.tab.o
  HOSTLD  scripts/kconfig/conf
scripts/kconfig/conf  --silentoldconfig Kconfig
Makefile:679: Cannot use CONFIG_CC_STACKPROTECTOR_STRONG: -fstack-protector-strong not supported by compiler
```

**Comment by tinxx** [comment]
The problem seems to be resolved in `grsec 3.1/201606262019`. Can you confirm?

**Comment by nning** [comment]
Yes, I can confirm this; let's close the issue then.

As a side note: I could not reproduce this problem with the core repository kernel.
